nodebuilder/share/types/get_range_result.go

```diff
@@ -0,0 +1,124 @@
+package share_types //nolint:revive
```

Comment:
one general comment re types made purely for the API -- maybe we should have package name include `api` to differentiate there? I'm not really sure what the best thing to do is but I feel like it should be differentiated from the genuine types we have defined in node.